jewishdiaspora/urls.py

```diff
@@ -11,6 +11,7 @@
     path('', views.HomeView.as_view(), name='home'),
     path('admin/', admin.site.urls),
     path('artifact/', include('artifacts.urls')),
+    url(r'^new/$', views.TheNewForm.as_view(), name='artifacts_donors_registration'),
```

Comment:
Next time use path instead of url, like the rest of the url patterns.
This is the new url's pattern in Django 2.